packages/contentful--cra-template-create-contentful-app/template/src/components/ConfigScreen.tsx

```diff
@@ -1,8 +1,8 @@
 import React, { useCallback, useState, useEffect } from 'react';
 import { AppExtensionSDK } from '@contentful/app-sdk';
 import { PlainClientAPI } from 'contentful-management';
-import { Heading, Form, Workbench, Paragraph } from '@contentful/forma-36-react-components';
-import styles from './ConfigScreen.module.css';
+import { Heading, Form, Paragraph, Flex } from '@contentful/f36-components';
+import { css } from 'emotion';
```

Comment:
`emotion` is not part of the template anymore. Please keep using the CSS module

Reply:
@andipaetzold `emotion` is a recommended way to style Forma v4 components and I think we need to re-evaluate the decision

Reply:
`emotion` was only used in this file and we saw that as an opportunity to get rid of a dependency and make the bundle smaller. If we want to reintroduce `emotion` with this PR, we'd need to add it back to `template.json`.

Reply:
I believe that we want to add it back as it's the recommended library to be used with forma v4 components, right @burakukula and @suevalov ??

Reply:
@massao yes, I'd do that